### DISCUSSION TITLE
Latest thumbnail?

### DISCUSSION THREAD

**Discussion by renarena** [blakeblackshear/frigate#3820] [Ask A Question]
I look for a way to get the latest thumbnail of an event to use in notifications.
I need it at time, not at end. If i get more then one notifys for one event it often has the same image, but the object has moved inbetween.

**Comment by NickM-27** [comment]
The thumbnail of an event is not continually updated, you'll need to use the latest snapshot of that's what you're after.

**Comment by renarena** [comment]
The snapshot is to big/the object in it to small. 
I need it for an smartphone notification. Its not helpful, if several notifications contain the same image. 
Its not needed to see what was, but what currently is, so you can take care of it.
So this is a feature request?
Can i do anything else? I set  best_image_timeout=10
